README.md

```diff
@@ -1 +1,29 @@
-# operate-first-workloads
\ No newline at end of file
+# operate-first-workloads
+
+Coordination of projects that intend to "Operate First" at the Mass Open Cloud (MOC).
+
+Operating First is about develpers bringing pre-releases of projects to a production
+cloud to vet their operation as well as their functionality. Only by operating in
+a production cloud and interacting with cloud operators can projects be sure that
+their code is ready for widespead deployment and use.
+
+These projects are the operate first workloads.
+
+Where appropriate, projects are encouraged to develop their operations in the 
+open here at Open Infrastructure Labs, alongside cloud operators that do the
+same, and where open source methods and principles are applied to operations.
+
+Coordination of these projects are documented here. There a live "scrum of scrums"
+meeting for representatives of these projects that is held bi-weekly via google meet 
+on Wednesdays, 8AM US Eastern time. Contact Bill Burns (bburns@redhat.com) 
+for an invite.
+
+Projects are expected to create a repo here with a brief description of the project's
```

Comment:
I think projects should create a document in this repo - or a page in the wiki. I don't think they should create their own repository